src/components/header/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { cookies } from 'next/headers';
+
+import Header from './header';
+
+export default function Page() {
```

Comment:
Did you mean Header() ?

Reply:
this is component

Reply:
I still don't understand what you mean, I just importing it " import Header from './header'" .
or you mean I have to call Header() instead of Page()?

Reply:
Yes, the naming is confuzing. It is still header component, so it should be like 
AuthorizedHeader 
Or
import BaseHeader from './header';
export default function Header